app/views/_swagger.njk

```diff
@@ -0,0 +1,49 @@
+{% extends "govuk/template.njk" %}
+{% block head %}
+    <!--[if !IE 8]><!-->
+    
+    <link href="/static/css/core.08d92743fd36535a3b5a.css" rel="stylesheet">
```

Comment:
I think these guids are re-generated each time a webpack build is done. You'll probably have to use the machnism like in _layout.template.njk to include these dynamic files